__tests__/articles.test.js

```diff
@@ -236,3 +236,84 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200 should return comments objects for the corresponding article_id", () => {
+    const votePatch = {
+      inc_votes: 42,
+    };
+    const article_id = 1;
+
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send(votePatch)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Object);
```

Comment:
No need to assert against the type, we are proving it's an object in two lines time :)